deploy/operator.yaml

```diff
@@ -111,4 +111,15 @@ spec:
       - name: kubelet-path
         hostPath:
           path: /var/lib/kubelet
-          type: Directory
\ No newline at end of file
+          type: Directory
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: seccomp-operator
+  name: default-profiles
+  annotations:
+    seccomp.security.kubernetes.io/profile: "true"
+data:
+  nginx-1.19.1.json: |-
```

Comment:
I have mentioned this in a few PR's, but we should think about making some of the deployment (such as the install of default profiles) configurable 👍

Reply:
I'll follow-up on that. I have no clue how we wanna do that without adding another level of abstraction. Do we wanna make it configurable via environment variables, Helm or a config file? I think a config file could be a good first step, which could be deployed in parallel to the deploy/operator.yaml. WDYT?